src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -193,6 +197,14 @@ public boolean isPinned()
     @Override
     public RestAction<Void> removeReaction(@Nonnull Emote emote, @Nonnull User user)
     {
+        Checks.notNull(user, "User");  // to prevent NPEs
+        // check if the passed user is the SelfUser, then the ChannelType doesn't matter
+        if (user.equals(getJDA().getSelfUser()))
+            // we can safely remove that
+            return channel.removeReactionById(getIdLong(), emote);
```

Comment:
nit: When the if body is more than one line it should use braces, the comment could be moved behind the return or use braces.

Reply:
I'd like to see this change made too.

Reply:
Let me change that really quick. :)